src/libraries/Microsoft.PowerFx.Connectors/ODataQueryableTableValue.cs

```diff
@@ -47,68 +47,70 @@ internal override TableValue FirstN(int n)
         }
     }
 
-    public readonly struct ODataParams
+    public readonly struct ODataParameters
     {
         // Missing parameters: skip, skipToken, expand, search, select, apply
-        private readonly bool _count;
-        private readonly string _filter;
-        private readonly string _orderby;
-        private readonly int _top;
+        public readonly bool Count;
```

Comment:
>public readonly bool Count; [](http://example.com/codeflow?start=8&length=27)

nit - fields shouldn't be public . Properties instead. Like:

public bool Count {get; private set; }